MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Tabs.xaml

```diff
@@ -0,0 +1,414 @@
+                                BorderBrush="{TemplateBinding BorderBrush}" 
+                                BorderThickness="0"
+                                Visibility="Hidden" RenderTransformOrigin="0.5,0.5">
+                            <!--<Border.RenderTransform>
+                                <ScaleTransform x:Name="ScaleTransform" ScaleX="0" ScaleY="1" />
+                            </Border.RenderTransform>-->
+                            <Rectangle x:Name="PART_BackgroundSelection" Fill="{TemplateBinding Background}" Opacity="0.12"/>
+                        </Border>
+                    </Grid>
+                    <ControlTemplate.Triggers>
+                        <Trigger Property="IsEnabled" Value="False">
+                            <Setter Property="Opacity" Value="0.23"/>
+                        </Trigger>
+                        <Trigger Property="IsSelected" Value="True">
+                            <Setter TargetName="contentPresenter" Property="Opacity" Value="1"/>
+                            <Setter TargetName="SelectionHighlightBsorder" Property="Visibility" Value="Visible" />
+                        </Trigger>
+                        <MultiTrigger>
+                            <MultiTrigger.Conditions>
+                                <Condition Property="IsSelected" Value="True"/>
+                                <Condition Property="wpf:ColorZoneAssist.Mode" Value="Standard"/>
+                            </MultiTrigger.Conditions>
+                            <Setter TargetName="contentPresenter" Property="Opacity" Value="1"/>
+                            <Setter TargetName="SelectionHighlightBsorder" Property="Visibility" Value="Visible" />
+                            <Setter TargetName="contentPresenter" Property="Foreground" Value="{DynamicResource PrimaryHueMidBrush}"/>
+                        </MultiTrigger>
+
+                        <!--<DataTrigger Binding="{Binding TabStripPlacement, RelativeSource={RelativeSource AncestorType={x:Type TabControl}}}" Value="Bottom">
+                            <Setter TargetName="SelectionHighlightBsorder" Property="BorderThickness" Value="0,2,0,0" />
+                        </DataTrigger>
+                        <DataTrigger Binding="{Binding TabStripPlacement, RelativeSource={RelativeSource AncestorType={x:Type TabControl}}}" Value="Left">
+                            <Setter TargetName="SelectionHighlightBsorder" Property="BorderThickness" Value="0,0,2,0" />
+                        </DataTrigger>
+                        <DataTrigger Binding="{Binding TabStripPlacement, RelativeSource={RelativeSource AncestorType={x:Type TabControl}}}" Value="Right">
+                            <Setter TargetName="SelectionHighlightBsorder" Property="BorderThickness" Value="2,0,0,0" />
+                        </DataTrigger>-->
+                        <Trigger Property="wpf:ColorZoneAssist.Mode" Value="PrimaryMid">
```

Comment:
Are these setters needed, or is the attached properties that are being used on the ColorZone sufficient. If we need these we should also expand them to handle the Custom mode as well.

Reply:
It's used to set the foreground color of elements, and the seletor rectangle of the selected tabitem (background property).